speech-synthesis/client.g.ts

```diff
@@ -161,7 +161,7 @@ namespace modules {
         */
         //% group="Speech synthesis"
         //% blockId=jacdac_speechsynthesis_speak_cmd
-        //% block="%speechsynthesis speak"
+        //% block="%speechsynthesis speak %text"
```

Comment:
@pelikhan why isn't the correct block signature auto-generated?

Reply:
Did you manually update this file? Your change will be erased on the next refresh.

Reply:
I did, and oh!

Reply:
Wouldn't it generate clientg.ts?

Reply:
There should be a comment at the some saying don't edit this file.
It's a bug in the generator.

Reply:
I mean to say: doesn't the generator generate clientg.ts?

Here I've modified client.g.ts?

I should move to client.ts then instead?